backend-stack-bubbling/app.py

```diff
@@ -244,13 +252,13 @@ def get():
 		'user_id': '$answers.user_id',
 		'is_best_answer': '$answers.is_best_answer'}}
 	 
-	 ])
-
+        ])
+        result = {
+            "AList": list(res),
```

Comment:
Meaningful name + Naming Convention 
"AList", "qVoteCount"

Reply:
These variable names are improved, now convention is being followed properly.